websharks-typinator-sets.rb

```diff
@@ -0,0 +1,43 @@
+require "formula"
+
+class WebsharksTypinatorSets < Formula
+  homepage "https://github.com/websharks/typinator-sets"
+  url "https://github.com/websharks/typinator-sets.git", :branch => "master"
+
+  depends_on "terminal-notifier" => :recommended
```

Comment:
I don't see any use of [terminal-notifier](https://github.com/alloy/terminal-notifier) in your formula, so you could remove this line if you like. I had in the OSA formula because some of the scripts that I wrote call upon this app.